lib/zold/commands/remote.rb

```diff
@@ -63,7 +63,9 @@ def run(args = [])
     #{Rainbow('remote elect').green}
       Pick a random remote node as a target for a bonus awarding
     #{Rainbow('remote trim').green}
-      Remote the least reliable nodes
+      Remove the least reliable nodes
+    #{Rainbow('remote select [number]').green}
```

Comment:
@botanicus it seems that this goes against our convention. Let's use `--max-nodes` as was suggested in https://github.com/zold-io/zold/issues/292